pulumi/volumeReaper/src/snapshot.ts

```diff
@@ -0,0 +1,47 @@
+import * as k8s from '@kubernetes/client-node'
+import { KubernetesObject } from '@kubernetes/client-node'
```

Comment:
we can always use `k8s.KubernetesObject` to avoid multiple imports of the same package. would chose one or the other, and stick with that for this package throughout.